wpilibj/src/main/java/edu/wpi/first/wpilibj/drive/DifferentialDrive.java

```diff
@@ -93,7 +93,10 @@
   private final SpeedController m_rightMotor;
 
   private boolean m_reported;
-
+  
+  /**
+   * Wheel speeds for a differential drive. Uses normalized voltage [-1.0..1.0].
```

Comment:
If you want the second sentence in another paragraph, add `<p>` before it. C++ doesn't need it for multiple paragraphs to work.